pom.xml

```diff
@@ -2393,17 +2435,6 @@
                 </rules>
               </configuration>
             </execution>
-            <execution>
```

Comment:
I had to remove this because the current approach relies on multiple `hadoop-client` dependencies when Hadoop profile is 2.7. Not sure if there is a way to exclude certain dependencies from this enforcer.